orquesta/tests/unit/conducting/test_workflow_conductor_performance.py

```diff
@@ -76,7 +76,7 @@ def test_serialization_function_of_data_size(self):
         conductor.deserialize(conductor.serialize())
 
     def test_runtime_function_of_splits_count(self):
-        num_tasks = 25
```

Comment:
we may need to change this back to 25 for our ci/cd testing.  the runtime is not that powerfule